rustls/src/server/tls13.rs

```diff
@@ -598,6 +598,7 @@ mod client_hello {
         common.send_msg(m, false);
     }
 
+    #[allow(clippy::needless_pass_by_ref_mut)] // cx only mutated if cfg(feature = "quic")
```

Comment:
The non-nightly clippy bails on this `allow` because the lint only exists in the nightly clippy right now. Probably need to deploy the same strategy as we used in rustls-ffi (https://github.com/rustls/rustls-ffi/commit/4f977fb49c5ca1cda6fa63dfafb68b6b245d3eae) and update the clippy invocation w/ `-A unknown-lints` at least temporarily :thinking:

Reply:
Thanks! Doing that now.